### PR TITLE
Add 3rd chunk of Cobra examples

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Adds the third chunk of Cobra examples to the cli help.
As we're putting together a release tomorrow, tried to
hit the heavy commands with this PR.

### CODE REVIEW

File: cmd/podman/pod_stop.go [hunk 1]
```diff
@@ -27,7 +27,9 @@ var (
 			podStopCommand.GlobalFlags = MainGlobalOpts
 			return podStopCmd(&podStopCommand)
 		},
-		Example: "POD-NAME [POD-NAME ...]",
+		Example: `podman pod stop mywebserverpod
+  podman pod stop 490eb 3557fb
+  podman pod stop --latest`,
```
Comment (mheon) [inline cmd/podman/pod_stop.go]: Have a --timeout 0 on one of these

File: cmd/podman/run.go [hunk 2]
```diff
@@ -30,7 +30,9 @@ var (
 			runCommand.GlobalFlags = MainGlobalOpts
 			return runCmd(&runCommand)
 		},
-		Example: "IMAGE [COMMAND [ARG...]]",
+		Example: `podman run imageID ls -alF /etc
+  podman run --net=host imageID dnf -y install java
+  podman run --volume /var/hostdir:/var/ctrdir --interactive --tty fedora /bin/bash`,
```
Comment (mheon) [inline cmd/podman/run.go]: I think we should swap this to `-t -i` - these are incredibly common options, and people will almost always see the short versions

File: cmd/podman/start.go [hunk 3]
```diff
@@ -28,7 +28,9 @@ var (
 			startCommand.GlobalFlags = MainGlobalOpts
 			return startCmd(&startCommand)
 		},
-		Example: "CONTAINER-NAME [CONTAINER-NAME ...]",
+		Example: `podman start mywebserver
+  podman start 860a4b231279 5421ab43b45
+  podman start --interactive --attach 860a4b231279`,
```
Comment (mheon) [inline cmd/podman/start.go]: Maybe a --latest example?

File: cmd/podman/pod_stop.go (hunk 1 above)
Comment (TomSweeneyRedHat) [reply cmd/podman/pod_stop.go]: added to the second

File: cmd/podman/run.go (hunk 2 above)
Comment (TomSweeneyRedHat) [reply cmd/podman/run.go]: will do.  I figured 98% of the folks would know -i -t, I spelled it out for the 2%.  But it is lengthy.

File: cmd/podman/start.go (hunk 3 above)
Comment (TomSweeneyRedHat) [reply cmd/podman/start.go]: Done

**Comment by TomSweeneyRedHat** [comment]
space are going to be the death of me yet.  I think I've all the gitvalidate/space issues cleared.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**